packages/dds/tree/src/feature-libraries/sequence-field/utils.ts

```diff
@@ -783,3 +823,84 @@ export function areComposable(changes: TaggedChange<Changeset<unknown>>[]): bool
     }
     return true;
 }
+
+export interface CrossFieldTable<T = unknown> extends CrossFieldManager<T> {
```

Comment:
There is also a CrossFieldTable type in modularChangeGamily, are both needed? Can one extend the other?
At least it would be less confusing if they had different names.

Reply:
I'd rather deal with this in a future PR.